### PR TITLE
Adding performance counters, fixing comment typo in logs

### PR DESCRIPTION
**Pull request by mrickard**
Signed-off-by: mrickard <mrickard@newrelic.com>

### CODE REVIEW

File: azureLogIngestion/adapter/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -1,5 +1,111 @@
...
+      "clientType": "PC",
+      "hostInstanceId": "{hostInstanceId}",
+      "instance": "??APP_WIN32_PROC??",
+      "processId": "4608",
+      "resourceId": "/SUBSCRIPTIONS/{subscriptionId}/RESOURCEGROUPS/{functionAppName}/PROVIDERS/MICROSOFT.INSIGHTS/COMPONENTS/{resourceGroupName}",
+      "sdkVersion": "af_azwapccore:2.16.0-18277",
+      "timestamp": 1621887508465,
+    },
+    "interval.ms": undefined,
+    "name": "% Processor Time",
```
Comment (kolanos) [inline azureLogIngestion/adapter/__snapshots__/index.test.ts.snap:22]: Isn't this a gauge and not a count?

File: azureLogIngestion/adapter/__snapshots__/index.test.ts.snap [hunk 2]
```diff
@@ -1,5 +1,111 @@
...
+      "clientIp": "0.0.0.0",
+      "clientType": "PC",
+      "hostInstanceId": "{hostInstanceId}",
+      "instance": "??APP_WIN32_PROC??",
+      "processId": "4608",
+      "resourceId": "/SUBSCRIPTIONS/{subscriptionId}/RESOURCEGROUPS/{functionAppName}/PROVIDERS/MICROSOFT.INSIGHTS/COMPONENTS/{resourceGroupName}",
+      "sdkVersion": "af_azwapccore:2.16.0-18277",
+      "timestamp": 1621887508465,
+    },
+    "interval.ms": undefined,
```
Comment (kolanos) [inline azureLogIngestion/adapter/__snapshots__/index.test.ts.snap:21]: Is there a default interval for performance counters we can set here?

**Review by kolanos — CHANGES_REQUESTED** [review]

File: azureLogIngestion/adapter/__snapshots__/index.test.ts.snap (hunk 2 above)
Comment (mrickard) [reply azureLogIngestion/adapter/__snapshots__/index.test.ts.snap:27]: Let's go with the 10000 you'd suggested as the default. Counters might have an interval property, but it's in the form of `\d+[ms|s|m]` (and probably typical cron values as well).

File: azureLogIngestion/adapter/__snapshots__/index.test.ts.snap (hunk 1 above)
Comment (mrickard) [reply azureLogIngestion/adapter/__snapshots__/index.test.ts.snap:28]: @kolanos It is a gauge and not a count...but apparently in the Node SDK, the gauge type inherits from a generic metric type. If the SDK creates a gauge metric without an intervalMs, it'll be defined on the metric as `undefined`.

File: azureLogIngestion/adapter/__snapshots__/index.test.ts.snap (hunk 2 above)
Comment (mrickard) [reply azureLogIngestion/adapter/__snapshots__/index.test.ts.snap:27]: I'm going to add a conversion method for these, but for right now I'm using the 10000 figure.

**Review by kolanos — APPROVED** [review]

**Pull request merged**